### PR TITLE
Add compiling of benchmarks to build sequence

### PR DESCRIPTION
**Pull request by myreen**
I'm curious to see if the regression test works. I've inserted the compilation of the benchmark examples right after the sexp bootstrap.

### CODE REVIEW

File: compiler/benchmarks/cakeml_benchmarks/cakeml/Holmakefile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+
```
Comment (xrchz) [inline compiler/benchmarks/cakeml_benchmarks/cakeml/Holmakefile]: why blank line?
Also I think I'd like to see `INCLUDES =` first, for consistency

File: compiler/benchmarks/mlton_benchmarks/cakeml/Holmakefile [hunk 2]
```diff
@@ -0,0 +1,20 @@
+
```
Comment (xrchz) [inline compiler/benchmarks/mlton_benchmarks/cakeml/Holmakefile]: (same as above)

File: compiler/benchmarks/cakeml_benchmarks/cakeml/Holmakefile [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+all: $(DEFAULT_TARGETS) README.md build-benchmarks
+.PHONY: all
+
+README_SOURCES = $(wildcard *Script.sml) $(wildcard *Lib.sml) $(wildcard *Syntax.sml)
+DIRS = $(wildcard */)
+README.md: $(CAKEMLDIR)/developers/readme_gen readmePrefix $(patsubst %,%readmePrefix,$(DIRS)) $(README_SOURCES)
```
Comment (xrchz) [inline compiler/benchmarks/cakeml_benchmarks/cakeml/Holmakefile]: missing `dprot`

File: compiler/benchmarks/cakeml_benchmarks/cakeml/Holmakefile [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+	$(protect $(CAKEMLDIR)/developers/readme_gen) $(README_SOURCES)
+
+cake.S: $(CAKEMLDIR)/unverified/sexpr-bootstrap/x64/64/cake.S
```
Comment (xrchz) [inline compiler/benchmarks/cakeml_benchmarks/cakeml/Holmakefile]: missing `dprot`

File: compiler/benchmarks/cakeml_benchmarks/cakeml/Holmakefile [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+	$(CP) $(CAKEMLDIR)/unverified/sexpr-bootstrap/x64/64/cake.S cake.S
```
Comment (xrchz) [inline compiler/benchmarks/cakeml_benchmarks/cakeml/Holmakefile]: missing `protect`

**Comment by myreen** [comment]
@xrchz I don’t disagree with any of your suggestions. I’m running short on time, so feel free to just commit the changes rather than writing comments.

**Pull request merged**